rules/linux/persistence_linux_group_creation.toml

```diff
@@ -0,0 +1,79 @@
+[metadata]
+creation_date = "2023/02/13"
+integration = ["system"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/03/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies attempts to create a new group. Attackers may create new groups to escalate privileges 
+or establish persistence on a system or domain.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-system.auth-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Linux Group Creation"
+note = """## Triage and analysis
+
+### Investigating Group Creation
+
+Attackers may create new groups to maintain access to victim systems or escalate privileges by assigning a compromised account to a privileged group.
+
+This rule identifies the usages of `groupadd` and `addgroup` to create new groups.
+
+#### Possible investigation steps
+
+- Investigate the process execution chain (parent process tree) for unknown processes. Examine their executable files for prevalence, whether they are located in expected locations, and if they are signed with valid digital signatures.
+- Identify the user account that performed the action and whether it should perform this kind of action.
+- Identify if a user account was added to this group after creation.
+- Investigate other alerts associated with the user/host during the past 48 hours.
```

Comment:
A step to enumerate the groups from the system can be added here, and a osquery too (probably)

Reply:
Added OSQuery queries.